src/main/scala/com/codacy/client/bitbucket/v2/service/RepositoryServices.scala

```diff
@@ -16,14 +16,27 @@ class RepositoryServices(client: BitbucketClient) {
   def getRepositories(
       ownerInfo: OwnerInfo,
       pageLength: Option[Int] = Option(100),
+      pageRequest: Option[PageRequest],
```

Comment:
@bmbferreira @machadoit @peavers should this have a default value to avoid changes when bumping the library? If so I would just move this parameter to the end of the list to really avoid any changes.

Reply:
yup, I would be ok with having none by default to fetch everything like we're doing before